MobileCenterDistribute/MobileCenterDistribute/MSDistribute.h

```diff
@@ -1,9 +1,9 @@
 #import "MSServiceAbstract.h"
 
 /**
- * Mobile Center updates service.
+ * Mobile Center Distribute service.
  */
-@interface MSUpdates : MSServiceAbstract
+@interface MSDistribute : MSServiceAbstract
 
 /**
  * Change The URL that will be used for generic update related tasks, e.g. fetching the auth token..
```

Comment:
This is for `setApiUrl` which only takes care of update (not install). Hard to say distribute here since distribute consists of install as well as update.